imports/wasi_snapshot_preview1/fs_test.go

```diff
@@ -1857,7 +1857,24 @@ var (
 		'a', 'b', '-', // name
 	}
 
-	dirents = append(append(append(append(direntDot, direntDotDot...), dirent1...), dirent2...), dirent3...)
+	// TODO: this entry is intended to test reading of a symbolic link entry,
```

Comment:
yeah or using a real file for this special case, until fstest.FS has symlink support. If you want to follow-up with a real file that's probably easiest/best alternative prior to some wider change